xrpl/asyncio/transaction/main.py

```diff
@@ -444,10 +443,10 @@ async def _calculate_fee_per_transaction_type(
         The expected Transaction fee in drops
     """
     # Reference Transaction (Most transactions)
-    if client is None:
-        net_fee = 10  # 10 drops
-    else:
-        net_fee = int(await get_fee(client))  # Usually 0.00001 XRP (10 drops)
+
+    net_fee = int(
+        await get_fee(client)
```

Comment:
thought we should default to 200 drops always? Is the new recommendation that we don't have a default fee anymore?

Reply:
No, we do not have a default value. I used 200 drops as an example value, it was chosen arbitrarily.

We will know the transaction fees once all the validators have reached a compromise. Here is more detail on the Fee Voting process: https://xrpl.org/docs/concepts/consensus-protocol/fee-voting#voting-process